Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -930,7 +930,9 @@
             "com.apple.iphone.axserver-systemwide"
             "com.apple.mobileassetd.v2"
             "com.apple.mobilegestalt.xpc"
+#if !ENABLE(GPU_PROCESS_BY_DEFAULT)
```

Comment:
Should we do the same for macOS where capture is either UIProcess or GPUProcess?